concourse/model/step.py

```diff
@@ -156,12 +156,14 @@ def __init__(
         self,
         name,
         is_synthetic=False,
+        always_notify_pull_requests=False,
```

Comment:
I don't think it is appropriate to add another formal parameter for this case. how about simply re-using the "is_synthetic" flag? That should do what we want, right?